lib/ntpUtil.js

```diff
@@ -29,6 +57,62 @@ const generatePublicKeyAndID = (privateKeyFile) => {
   }
 }
 
+const isSupportedSchemaVersion = (version) => {
+  return version === 1 || version === 2;
+}
+
+const prepareAssets = (jsonFileUrl, targetResourceDir, targetJsonFileName) => {
+  return new Promise(function (resolve, reject) {
+    let jsonFileBody = '{}'
+
+    // Download and parse jsonFileUrl.
+    // If it doesn't exist, create with empty object.
+    request(jsonFileUrl, async function (error, response, body) {
+      if (error) {
+        console.error(`Error from ${jsonFileUrl}:`, error)
+        return reject(error)
+      }
+      if (response && response.statusCode === 200) {
+        jsonFileBody = body
+      }
+      let photoData = {}
+      try {
+        photoData = JSON.parse(jsonFileBody)
+      } catch (err) {
+        console.error(`Invalid json file ${jsonFileUrl}`)
+        return reject(error)
+      }
+      // Make sure the data has a schema version so that clients can opt to parse or not
+      const incomingSchemaVersion = photoData.schemaVersion
```

Comment:
maybe this can be `const incomingSchemaVersion = photoData.schemaVersion || 1` to let it fall back to 1, if it's missing? WDYT @petemill

Reply:
Changed to set default schemaVersion if it's empty. (Back to original code :) )

Reply:
Nice! LGTM